client/src/app/dashboard/datatable-notification/datatable-notification.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, Inject, OnInit, ViewChild} from '@angular/core';
```

Comment:
Inject and MAT_DIALOG_DATA are currently unused imports, please make sure to check if they are necessary and remove them otherwise.

Reply:
done